src/device/dnomem.c

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) 2023 Vojtech Horky
+ * All rights reserved.
+ *
+ * Distributed under the terms of GPL.
+ *
+ *
+ * Debug no-memory device. Writing to this area causes simulator to enter
+ * interactive mode.
+ *
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <inttypes.h>
+#include "dnomem.h"
+#include "../fault.h"
+#include "../utils.h"
+
+/** Access handling */
+enum nomem_access_handling_e {
+	NOMEM_ACCESS_WARN,
+	NOMEM_ACCESS_ENTER_DEBUGGER,
```

Comment:
Perhaps call this `NOMEM_ACCESS_BREAK`? We are not really entering a debugger, but `msim` interactive mode.

Reply:
Done.